ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -381,8 +381,6 @@ tests:
   optional: true
   steps:
     cluster_profile: aws
-    env:
-      AWS_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
```

Comment:
@r4f4 I think the mini-perm test it's not necessary for UPI, as the list for IPI only.

Btw, the`upi-install-aws-cluster` is not adapted to the mini-perm test, it uses  `${CLUSTER_PROFILE_DIR}/.awscred` always.